### PR TITLE
feat: Curve BTC Bridge

### PR DESCRIPTION
**Pull request by dekz**
https://github.com/0xProject/0x-monorepo/pull/2633

# Description

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -283,7 +283,7 @@ export const GAS_SCHEDULE_V0: FeeSchedule = {
     [ERC20BridgeSource.Eth2Dai]: () => 5.5e5,
     [ERC20BridgeSource.Kyber]: () => 8e5,
     [ERC20BridgeSource.Curve]: fillData => {
-        switch ((fillData as CurveFillData).poolAddress.toLowerCase()) {
+        switch ((fillData as CurveFillData).curve.poolAddress.toLowerCase()) {
```
Comment (dorothy-zbornak) [inline src/config.ts:286]: We also need to add estimates for `0x93054188d876f558f4a66b2ef1d97d16edf0895b` and `0x7fc77b5c7614e1533320ea6ddc2eb61fa00a9714`. These are about 600k on v0.

Comment (dekz) [reply src/config.ts:286]: done

**Comment by dekz** [comment]
deploy staging

**Comment by dekz** [comment]
deploy production

**Pull request merged**